_data/simple-icons.json

```
@@ -2253,6 +2253,11 @@
             "hex": "80B5E3",
             "source": "https://chocolatey.org/media-kit"
         },
+        {
+            "title": "Chromatic",
+            "hex": "FC521F",
+            "source": "https://www.chromatic.com/company/about"
```

```suggestion
            "source": "https://www.chromatic.com"
```

The about page does not contain additional material other than the header logo. Let's use the homepage link instead.

---

Updated as per your suggestion.